src/lib/stepper/step-label.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Directive, TemplateRef} from '@angular/core';
+import {CdkStepLabel} from '@angular/cdk';
+
+@Directive({
+  selector: '[md-step-label], [mat-step-label]',
+})
+export class MdStepLabel extends CdkStepLabel {
+  constructor(public template: TemplateRef<any>) {
```

Comment:
I don't _think_ you need the `public` here since it's on the super class (but I could be wrong)